fcomm/examples/Makefile

```diff
@@ -1,8 +1,17 @@
 build : ../../target/release/fcomm
 	cargo build --release
 
+all: eval-fibonacci fibonacci-claim.json fibonacci-proof.json fibonacci-proof2.json fibonacci-proof fibonacci-proof2 verify-fibonacci-proof verify-fibonacci-proof2 num-list-commitment num-list-opening num-list-opening.json verify-num-list-opening chained-commitment chained-opening chained-opening.json verify-chained-opening chained-opening2 chained-opening2.json verify-chained-opening2
```

Comment:
Nit: you can break this in several lines by escaping the linebreak character (ending the line with `\`).

Reply:
done